js/app/models/calendarmodel.js

```diff
@@ -90,7 +90,7 @@ app.factory('Calendar', function($window, Hook, VEventService, TimezoneService,
 
 				return Promise.all(promises).then(() => {
 					callback(vevents);
-					fcAPI.reportEventChange();
+					fcAPI.eventManager.currentPeriod.release();
```

Comment:
This is still quite hacky, but I hope we can get https://github.com/fullcalendar/fullcalendar/issues/1672 soon